### ISSUE TITLE
Allow users to fine-tune their availability

### ISSUE THREAD

**Issue by malini** [thunderbird/appointment#422]
**Is your feature request related to a problem? Please describe.**
Right now, you can only select a single time-frame and what days to apply it on. There isn't a way to block out certain days, or shift availability on certain days vs others, or designate timeslots in a day that you're willing to be booked

**Describe the solution you'd like**
We should work with design to incorporate an easy-to-use configuration form for variable availability on a more granular level. Details on what to fine-tune are TBD.

**Comment by ryanleesipes** [comment]
Agreed. I'm just not using Fridays right now because I can't mark the morning only for that day. 🤷

**Comment by malini** [comment]
Assigning this to myself to do a little competitive analysis soon

**Comment by laurelterlesky** [comment]
Design is working on this. First pass at this ticket for Beta 3 will be:
- Required: Check boxes for each day, so user can easily select which days they want
- Explore unlimited Interval support
- Explore copying availability and pasting into another day of the week

Out-of-Scope - date specific overrides (we'll look at this during MVP)

**Comment by solangevalverde** [comment]
Wireframe delivery 💌📬 

![Image](https://github.com/user-attachments/assets/0d7b6f89-bde6-4a00-bdfe-bcad27e3a956)

**Comment by devmount** [comment]
FYI: The first version of this is on stage, where you can set one custom availability per day. I'm now working on the next iteration to provide multiple slots per day.

**Comment by devmount** [comment]
@malini @laurelterlesky The following topics / edge cases that were found during review or just implemented by me should be discussed / decided. I'm just throwing them in here, so they don't get lost. There might come more.

1. Update of the dashboard calendar "availability preview entries" isn't implemented yet (since we wanted to remove the calendar iirc). So those entries won't update when setting fine-tune availability entries.
![Image](https://github.com/user-attachments/assets/c503acec-98ec-4708-b4e8-e3f757ac89c0)
2. Removal of the last availability entry for a weekday disables that day.
![Image](https://github.com/user-attachments/assets/33cee884-e591-499c-aa33-821c868f45f8)
3. Availability can be added until 11pm (for validation reasons, but that can be changed). Do we want to add slots until midnight?
![Image](https://github.com/user-attachments/assets/d229fe2b-3ffb-460b-9191-dc9b86f5d193)
4. The copy function shows every weekday as disabled that was already disabled in the configuration as well as the day that serves as the copy source.
5. The "Add slot" from the wireframe was replaced by me with a plus icon. There are strange languages (like e.g. German 😂 ) out there, that have really long words for stuff which wouldn't fit in there 😅 ("Add slot" would translate to "Zeitfenster hinzufügen" in German)

**Comment by malini** [comment]
we've implemented and deployed most of these changes. We can file any further development as follow-ups
